README.md

```diff
@@ -63,165 +63,171 @@ For the accelerator-specific features, you can install them by appending `#egg=o
+  # Create a ONNX Runtime Trainer
+- trainer = Trainer(
++ trainer = ORTTrainer(
+      model=model,
+      args=training_args,
+      train_dataset=train_dataset,
++     feature="sequence-classification", # The model type to export to ONNX
+      ...
+  )
+
+  # Use ONNX Runtime for training!
+  trainer.train()
+```
 
-cls_pipeline = pipeline("text-classification", model=model, tokenizer=tokenizer)
 
-results = cls_pipeline("I love burritos!")
-```
+### Accelerated inference
 
-Similarly, you can apply static quantization by simply setting `is_static` to `True` when instantiating the `QuantizationConfig` object:
+#### ONNX Runtime
 
-```python
-qconfig = AutoQuantizationConfig.arm64(is_static=True, per_channel=False)
-```
+To accelerate inference with ONNX Runtime, 🤗 Optimum uses _configuration objects_ to define parameters for optimization. These objects are then used to instantiate dedicated _optimizers_ and _quantizers_.
 
-Static quantization relies on feeding batches of data through the model to estimate the activation quantization parameters ahead of inference time. To support this, 🤗 Optimum allows you to provide a _calibration dataset_. The calibration dataset can be a simple `Dataset` object from the 🤗 Datasets library, or any dataset that's hosted on the Hugging Face Hub. For this example, we'll pick the [`sst2`](https://huggingface.co/datasets/glue/viewer/sst2/test) dataset that the model was originally trained on:
+Before applying quantization or optimization, first export our model to the ONNX format:
 
 ```python
-from functools import partial
-from optimum.onnxruntime.configuration import AutoCalibrationConfig
-
-# Define the processing function to apply to each example after loading the dataset
-def preprocess_fn(ex, tokenizer):
-    return tokenizer(ex["sentence"])
-
-# Create the calibration dataset
-calibration_dataset = quantizer.get_calibration_dataset(
-    "glue",
-    dataset_config_name="sst2",
-    preprocess_function=partial(preprocess_fn, tokenizer=tokenizer),
-    num_samples=50,
-    dataset_split="train",
-)
-# Create the calibration configuration containing the parameters related to calibration.
-calibration_config = AutoCalibrationConfig.minmax(calibration_dataset)
-# Perform the calibration step: computes the activations quantization ranges
-ranges = quantizer.fit(
-    dataset=calibration_dataset,
-    calibration_config=calibration_config,
-    operators_to_quantize=qconfig.operators_to_quantize,
-)
-
-# Apply static quantization on the model
-quantizer.quantize(
-    save_dir=save_directory,
-    calibration_tensors_range=ranges,
-    quantization_config=qconfig,
-)
+>>> from optimum.onnxruntime import ORTModelForSequenceClassification
+>>> from transformers import AutoTokenizer
+>>> model_checkpoint = "distilbert-base-uncased-finetuned-sst-2-english"
+>>> save_directory = "tmp/onnx/"
+>>> # Load a model from transformers and export it to ONNX
+>>> tokenizer = AutoTokenizer.from_pretrained(model_checkpoint)
```

Comment:
There should be `>>> ort_model = ORTModelForSequenceClassification.from_pretrained(model_checkpoint, from_transformers=True)` right above this right?

Reply:
Well spotted - not sure what happened with that commit!

Fixed in [a25bbd9](https://github.com/huggingface/optimum/pull/395/commits/a25bbd94cee2f90e4a7d7f92e4b80ab92f32dd31)